jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/file/ErrorManageUtil.java

```diff
@@ -145,4 +147,35 @@ public static Map<String,Object> formErrorCode() {
         log.info("Return Info {}",errorCode);
         return errorCode;
     }
+
+	public static void omitOpsAlert() {
+		for(Map.Entry<String,List<ErrorVH>> entry:errorMap.entrySet())
+		{
+			List<ErrorVH> errorVHList=entry.getValue();
+			for(ErrorVH errorVH:entry.getValue())
+			{
+				if(errorVH.getType().equals("phd")&&(errorVH.getException().contains("AccessDeniedException")||errorVH.getException().contains("SubscriptionRequiredException")))
+				{
+					log.info("Omit exception :{}",errorVH.getException());
+					errorVHList.remove(errorVH);
+				}
+				if(errorVH.getType().equals("kms")&&errorVH.getException().contains("AccessDeniedException"))
```

Comment:
3 blocks can be merged to one. 
errorVH.getException().contains("AccessDeniedException") && Arrays.asList("kms","s3","check").contains(errorVH.getType())